docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md

```diff
@@ -24,6 +24,10 @@ You must perform the following steps _while still in Kubernetes v1.24_:
 
 1. Map your active PSPs to Pod Security Standards:
     1. See which PSPs are still active in your cluster:
+       :::caution
+       This strategy may miss workloads that are not currently running, such as CronJobs, workloads that scale to zero, or workloads that have not rolled out yet.
```

Comment:
`workloads that scale to zero` - is this referring to workloads that are currently scaled down to zero?